plugin-maven/src/main/java/com/diffplug/spotless/maven/AbstractSpotlessMojo.java

```diff
@@ -88,6 +89,9 @@ public abstract class AbstractSpotlessMojo extends AbstractMojo {
 	@Component
 	private ResourceManager resourceManager;
 
+	@Component
+	protected BuildContext buildContext;
```

Comment:
```suggestion
	private BuildContext buildContext;
```

Reply:
Not possible as accessed by `SpotlessApplyMojo`.